lib/generators/revise_auth/model_generator.rb

```diff
@@ -1,32 +1,28 @@
 module ReviseAuth
   module Generators
-    class ModelGenerator < Rails::Generators::NamedBase
-      include Rails::Generators::ResourceHelpers
-
-      desc "Generates a model for authentication, default User"
+    class ModelGenerator < Rails::Generators::Base
+      desc "Generates the User model for authentication"
 
       source_root File.expand_path("templates", __dir__)
 
-      argument :name, required: false, default: "User"
```

Comment:
@patriciomacadden I think the goal was to make this work with any model name and default to 'User'

```rb
rails g revise_auth:model Account first_name last_name
```

Reply:
We decided it was significantly simpler if we only support `User` going forward and then don't have to deal with metaprogramming everything because we don't know the model name.

Reply:
Make sense @excid3